### PR TITLE
GCC 11 build fixes

### PR DESCRIPTION
**Pull request by villevoutilainen**

### CODE REVIEW

File: code/AssetLib/AMF/AMFImporter_Postprocess.cpp [hunk 1]
```diff
@@ -329,8 +329,8 @@ void AMFImporter::Postprocess_AddMetadata(const AMFMetaDataArray &metadataList,
     sceneNode.mMetaData = aiMetadata::Alloc(static_cast<unsigned int>(metadataList.size()));
     size_t meta_idx(0);
 
-    for (const AMFMetadata &metadata : metadataList) {
-        sceneNode.mMetaData->Set(static_cast<unsigned int>(meta_idx++), metadata.Type, aiString(metadata.Value));
+    for (const AMFMetadata *metadata : metadataList) {
+        sceneNode.mMetaData->Set(static_cast<unsigned int>(meta_idx++), metadata->Type, aiString(metadata->Value));
```
Comment (villevoutilainen) [inline code/AssetLib/AMF/AMFImporter_Postprocess.cpp:333]: error: loop variable ‘metadata’ of type ‘const AMFMetadata&’ binds to a temporary constructed from type ‘AMFMetadata* const’ [-Werror=range-loop-construct]
  332 |     for (const AMFMetadata &metadata : metadataList) {

File: code/Common/Importer.cpp [hunk 2]
```diff
@@ -78,6 +78,7 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #include <assimp/Profiler.h>
 #include <assimp/commonMetaData.h>
 
+#include <exception>
```
Comment (villevoutilainen) [inline code/Common/Importer.cpp:81]: GCC 11 has reorganized the stdlib header dependencies, so std::exception_ptr no longer comes in transitively, need to add these.

**Review by kimkulling — APPROVED** [review]

**Comment by kimkulling** [comment]
Looks good. I will merge it!

**Comment by kimkulling** [comment]
Merged, thanks a lot for your contribution.

**Pull request merged**